### DISCUSSION TITLE
SOAP API support for creating Case and CaseComment with EmailHeader fields

### DISCUSSION THREAD

**Discussion by phof** [jsforce/jsforce#1470] [Ideas]
Issue
REST API doens't provide a way to set EmailHeader values when creating a Case or a CaseComment.

Request
Add a `createCase` and `createCaseComment` methods to the SoapApi class to create a Case and CaseComment records via SOAP API, allowing to specify the EmailHeader fields documented at https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_header_emailheader.htm

**Comment by erikhopp** [comment]
I know this is an old issue, but I just wanted to +1 this as I think there is no way to enable the 'triggerAutoResponseEmail' parameter in the EmailHeader as specified here: https://help.salesforce.com/s/articleView?id=000325741&type=1 

Being able to enable this option would allow customer confirmation emails to be sent to a submitter when a new Case is programmatically created with the API. [https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_header_emailheader.htm]
